readme.md

```diff
@@ -32,6 +32,16 @@ $ composer require silverstripe/userforms
 
 You'll also need to run `dev/build`. You should see a new page type in the CMS 'User Defined Form'. This has a new 'Form' tab which has your form builder.
 
+## SPAM protection
+
+This module does not include SPAM protection out of the box. Consider installing `silverstripe/spamprotection` as well. This will allow CMS users to add a "SPAM Protection Field" to user forms.
+
+```sh
+$ composer require silverstripe/userforms
+```
```

Comment:
```suggestion
```
Is this line supposed to be here?

Reply:
Oups ... copy-pasted the line above and forgot to change the module name.

Reply:
Fixed